### DISCUSSION TITLE
Best Practice for arbitrarily ignoring certain updates?

### DISCUSSION THREAD

**Discussion by chris-dura** [renovatebot/renovate#8953] [Request Help] [answered]
What is the best practice for ignoring MRs for packages you DO NOT want to upgrade (usually `MAJOR` versions)... but you will eventually?

Here is an example of what I mean...

- My package.json currently has `"sass-loader": "^8.0.0"`
- The latest `sass-loader` is `11.x`
- `sass-loader@11.x` wants a minimum version of `webpack@5.x`
- However, a lot of the other packages I use want to stay on `webpack@4.x`
- So, I don't really want to update `sass-loader` _just_ because there's a newer version
- I'd much rather wait until I _know_ the other "more important" packages work with `webpack@5` and won't break...

So, currently, I'm just manually upgrading dependencies, so I just keep a log of which dependencies I _do not_ want to update for one reason or another. Since (I assume) Renovate is going to create an MR for `sass-loader` to update to `11.x`... having an MR sitting there is going to be a lot more tempting for a Maintainer to possibly merge it, without understanding that we actually DID NOT want to upgrade that... yet.

The docs are expansive, so I wasn't sure if there was an elegant way to "manage" this in Renovate (beyond just using `ignoreDeps`)?

I guess I'm looking for an "ignore until something else happens" functionality... like a way to configure Renovate to "only open MRs for dependencies that have compatible sub-dependencies in the same `MAJOR` version range"??

If I just use `ignoreDeps`... any suggestions on a way to document WHY those are ignored for future Maintainers? (other than just dropping it in the README.md 😆)

**Comment by viceice** [comment, marked as answer]
Use dependency dashboard approval.

So you see all updates in one issue and can approve when you are ready.

Approval requirement can be assigned via package rules.

**Reply by rarkins** [reply]
Instead of ignoreDeps you could write a package rule for each and use the "description" field. Check out this rule we use as an example of something similar: https://github.com/renovatebot/.github/blob/cf278491c92f78960db271c34d58522051aedfc3/default.json#L19-L25

**Reply by chris-dura** [reply]
Thanks, the `description` field is a nice way to document "inline"!
